### PR TITLE
JIT: catch illegal instruction errors

### PR DESCRIPTION
**Pull request by FioraAeterna**
Still crash, but at least give a message informing the world that it happened.

### CODE REVIEW

**Comment by waddlesplash** [comment]
LGTM

File: Source/Core/Core/PowerPC/PPCAnalyst.cpp [hunk 1]
```diff
@@ -673,6 +673,11 @@ u32 PPCAnalyzer::Analyze(u32 address, CodeBlock *block, CodeBuffer *buffer, u32
 			num_inst++;
 			memset(&code[i], 0, sizeof(CodeOp));
 			GekkoOPInfo *opinfo = GetOpInfo(inst);
+			if (!opinfo)
+			{
+				PanicAlert("Invalid PowerPC opcode: %x.", inst.hex);
+				Crash();
```
Comment (degasus) [inline Source/Core/Core/PowerPC/PPCAnalyst.cpp]: iirc the PanicAlert is able to crash itself, so this Crash() call isn't needed imo.

Comment (phire) [inline Source/Core/Core/PowerPC/PPCAnalyst.cpp]: I'm pretty sure PanicAlert doesn't crash itself.

Comment (BhaaLseN) [inline Source/Core/Core/PowerPC/PPCAnalyst.cpp]: ```PanicAlert``` is not a Yes/No dialog, so I have to agree with phire.

**Comment by magumagu** [comment]
It would be nice to generate a Program Exception like we're supposed to... but this is fine.

**Pull request merged**